workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/context/ListViewContext.tsx

```diff
@@ -305,18 +307,51 @@ const ListViewContextProvider: React.FC<
 	);
 
 	useEffect(() => {
-		if (!filter && filterSchemaStorage && filterPinnedStorage) {
+		const filters =
+			filterPinnedStorage?.filter &&
+			Object.keys(filterPinnedStorage?.filter).map((key) => {
+				if (Array.isArray(filterPinnedStorage?.filter[key])) {
+					return {
+						name: key,
+						value: (filterPinnedStorage?.filter as any)[key].map(
+							(options: Option) => options.value || options
```

Comment:
This part seems a bit strange, due to the any. But if it is the only way that made it work, for me is okay.

Reply:
@rafaelfs98